lib/scss/6_components/_conversations.scss

```diff
@@ -460,7 +460,7 @@ $conversation-topic-display: none !default;
     width: 100% !important;
     z-index: 3;
     position: relative;
-    padding: $spacing-unit-small;
+    padding: 50px 40px;
```

Comment:
Queria evitar ao máximo esses números mágicos em CSS. Isso resolve o problema do componente no tema padrão, mas fica insustentável para gerenciar temas e alterações mais globais. Minha sugestão é tentar usar um dos $spacing-unit-(tiny|small|normal|large|huge) quando for especificar espaçamentos no css.

Isso aparece em várias outras partes no código.